src/input/config.rs

```diff
@@ -117,6 +117,7 @@ pub struct NicConfig {
 #[derive(Clone, PartialEq)]
 pub struct FileConfig {
     pub ess: Essential,
+    pub allowed_extensions: Vec<String>,
 }
```

Comment:
@jinjoolee07 @sophie-cluml 

Is this supposed to include `.` before an extension? I think some documentation on it should be provided.

Reply:
@jinjoolee07 I would like to suggest creating an issue for the documentation for this and assigning yourself. I assume the PR that would resolve the issue would include a documentation comment right above this field.